builtin/common/misc_helpers.lua

```diff
@@ -312,12 +312,11 @@ function core.formspec_escape(text)
 	return text
 end
 
-
-function core.wrap_text(text, max_length, as_table)
+function core.line_break function(text, max_length)
```

Comment:
you still haven't fixed the obvious syntax error here
you should test that your PR works correctly